### PR TITLE
Ports "Revives "Adds sound effects to surgeries"" and others

### PR DESCRIPTION
**Pull request by nmajask**
# Document the changes in your pull request

Ports some surgery sound PRs and tweaks a few other surgery/sound related things. The PRs that this ports are:

-  ["Revives "Adds sound effects to surgeries"" #65578](https://github.com/tgstation/tgstation/pull/65578) by [maxspells](https://github.com/maxspells) and [nmajask](https://github.com/nmajask)
-  ["Surgery sound fix attempt 2" #66018](https://github.com/tgstation/tgstation/pull/66018) by [TheBoondock](https://github.com/TheBoondock)
-  ["Makes cautery a lot quieter" #66330](https://github.com/tgstation/tgstation/pull/66330) by [maxspells](https://github.com/maxspells)
-  ["Fixes a copy+paste issue with "open the hatch" surgery step sound" #66441](https://github.com/tgstation/tgstation/pull/66441) by [MrMelbert](https://github.com/MrMelbert)

# Spriting
No new sprites.

# Wiki Documentation

No changes needed. 

# Changelog

:cl:  maxspells, TheBoondock, MrMelbert, nmajask
soundadd: added sounds to surgery steps
bugfix: fixed medical sounds for wounds, rad laser, and PDA health scan
tweak: doctors now say "Clear!" during revive surgery 
spellcheck: fixed revive surgery saying "shock body" instead of "shock brain"  
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -1025,6 +1026,9 @@ GLOBAL_LIST_EMPTY(PDAs)
 		switch(scanmode)
 
 			if(PDA_SCANNER_MEDICAL)
+				if(beep_cooldown<world.time)
+					playsound(src, 'sound/effects/fastbeep.ogg', 20)
+					beep_cooldown = world.time+40
```
Comment (JamieD1) [inline code/game/objects/items/devices/PDA/PDA.dm]:
```suggestion
				if(beep_cooldown < world.time)
					playsound(src, 'sound/effects/fastbeep.ogg', 20)
					beep_cooldown = world.time + 40
```

File: code/game/objects/items/devices/traitordevices.dm [hunk 2]
```diff
@@ -82,6 +82,9 @@ effective or pretty fucking useless.
 	if(!irradiate || !is_syndicate(user)) // only syndicates are aware of this being a rad laser and know how to use it.
 		return
 	if(!used)
+		if(beep_cooldown<world.time)
+			playsound(src, 'sound/effects/fastbeep.ogg', 20)
+			beep_cooldown = world.time+40
```
Comment (JamieD1) [inline code/game/objects/items/devices/traitordevices.dm]:
```suggestion
		if(beep_cooldown < world.time)
			playsound(src, 'sound/effects/fastbeep.ogg', 20)
			beep_cooldown = world.time + 40
```

File: code/modules/surgery/mechanic_steps.dm [hunk 3]
```diff
@@ -27,16 +32,21 @@
...
 			"[user] begins to screw the shell of [target]'s [parse_zone(target_zone)].")
 
 /datum/surgery_step/mechanic_close/tool_check(mob/user, obj/item/tool)
-	if(implement_type == /obj/item && !tool.is_sharp())
-		return FALSE
+	if(istype(tool))
+		if(!tool.is_sharp())
+			return FALSE
+		if(tool.usesound)
+			preop_sound = tool.usesound
```
Comment (JamieD1) [inline code/modules/surgery/mechanic_steps.dm:50]:
```suggestion
			preop_sound = tool.usesound
```

File: code/modules/surgery/mechanic_steps.dm [hunk 4]
```diff
@@ -59,33 +72,57 @@
...
 			"[user] begins to unwrench some bolts in [target]'s [parse_zone(target_zone)].",
 			"[user] begins to unwrench some bolts in [target]'s [parse_zone(target_zone)].")
 
+
+/datum/surgery_step/mechanic_unwrench/tool_check(mob/user, obj/item/tool)
+	if(tool.usesound)
+		preop_sound = tool.usesound
+
+	return TRUE
+
```
Comment (JamieD1) [inline code/modules/surgery/mechanic_steps.dm]:
```suggestion
	return TRUE

```

File: code/modules/surgery/mechanic_steps.dm [hunk 5]
```diff
@@ -59,33 +72,57 @@
...
 	display_results(user, target, span_notice("You begin to wrench some bolts in [target]'s [parse_zone(target_zone)]..."),
 			"[user] begins to wrench some bolts in [target]'s [parse_zone(target_zone)].",
 			"[user] begins to wrench some bolts in [target]'s [parse_zone(target_zone)].")
 
+
+/datum/surgery_step/mechanic_wrench/tool_check(mob/user, obj/item/tool)
+	if(tool.usesound)
+		preop_sound = tool.usesound
+
+	return TRUE
```
Comment (JamieD1) [inline code/modules/surgery/mechanic_steps.dm]:
```suggestion
	return TRUE
```

File: code/modules/surgery/mechanic_steps.dm [hunk 6]
```diff
@@ -59,33 +72,57 @@
...
 		"[user] begins to open the hatch holders in [target]'s [parse_zone(target_zone)].",
-		"[user] begins to open the hatch holders in [target]'s [parse_zone(target_zone)].")
\ No newline at end of file
+		"[user] begins to open the hatch holders in [target]'s [parse_zone(target_zone)].")
+
+/datum/surgery_step/open_hatch/tool_check(mob/user, obj/item/tool)
+	if(tool.usesound)
+		preop_sound = tool.usesound
+
+	return TRUE
```
Comment (JamieD1) [inline code/modules/surgery/mechanic_steps.dm]:
```suggestion
	return TRUE
```

File: code/modules/surgery/surgery_step.dm [hunk 7]
```diff
@@ -76,8 +94,10 @@
 	var/user_speed_mod = 1
 
 	if(preop(user, target, target_zone, tool, surgery) == -1)
-		surgery.step_in_progress = 0
-		return
+		surgery.step_in_progress = FALSE
+		return FALSE
+
```
Comment (JamieD1) [inline code/modules/surgery/surgery_step.dm]:
```suggestion
```

**Review by JamieD1 — CHANGES_REQUESTED** [review]

**Review by JamieD1 — APPROVED** [review]

**Comment by VaelophisNyx** [comment]
hey about this, the sounds are like, excessively loud for what they are

**Pull request merged**